snapcraft/__init__.py

```diff
@@ -115,7 +115,11 @@
 import os
 import shutil
 
+from snapcraft._help import topic_help  # noqa
+from snapcraft._store import login, logout, upload  # noqa
```

Comment:
I can give you a personal note here too, I don't like the way tupling looks either :stuck_out_tongue: